plan.py

```diff
@@ -33,12 +47,100 @@
 saver = tf.train.Saver()
 saver.restore(sess, 'model/model.ckpt')
 
+if args.use_openvino:
+    import subprocess
+    import sys
+    import mo_tf
+    import os
+
+    from collections import namedtuple
+    from openvino.inference_engine import IECore
+    ie = IECore()
+
+    # Get input nodes names
+    exp_inp_node_x_name = expansion_net.X.name.split(':')[0]
+    exp_inp_node_k_name = expansion_net.k.name.split(':')[0]
+    roll_inp_node_name = rollout_net.X.name.split(':')[0]
+
+    pb_model_path = 'model.pb'
+    model_xml = 'model.xml'
+    model_bin = 'model.bin'
+
+    # EXPANSION_NET:
+    # Save frozen graph
+    input_node_names = [exp_inp_node_x_name, exp_inp_node_k_name]
+    output_node_names = ['TopKV2']
+
+    graph_def = tf.graph_util.convert_variables_to_constants(
+            sess,
+            sess.graph_def,
+            output_node_names)
+
+    from tensorflow.python.tools import optimize_for_inference_lib
+    graph_def = optimize_for_inference_lib.optimize_for_inference(
+            graph_def, input_node_names, output_node_names, tf.float32.as_datatype_enum)
+
+    with tf.io.gfile.GFile(pb_model_path, 'wb') as f:
+        f.write(graph_def.SerializeToString())
+
+    # Convert to OpenVINO IR
+    subprocess.run(
+        [
+            sys.executable, mo_tf.__file__, '--input_model', pb_model_path,
+            '--input', ','.join(input_node_names), '--input_shape', "[1, 10000],[1]",
+            '--freeze_placeholder_with_value', "Placeholder_2->5"
+        ],
+        check=True)
+
+    exp_net = ie.read_network(model=model_xml, weights=model_bin)
+    exp_exec_net = ie.load_network(network=exp_net, device_name='CPU')
+
+    os.remove(pb_model_path)
+    os.remove(model_xml)
+    os.remove(model_bin)
+
+
+    #ROLLOUT_NET:
+    # Save frozen graph
+    input_node_names = [roll_inp_node_name]
+    output_node_names = ['ArgMax']
+    
+    graph_def = tf.graph_util.convert_variables_to_constants(
+            sess,
+            sess.graph_def,
+            output_node_names)
+
+    from tensorflow.python.tools import optimize_for_inference_lib
+    graph_def = optimize_for_inference_lib.optimize_for_inference(
+            graph_def, input_node_names, output_node_names, tf.float32.as_datatype_enum)
+
+    with tf.io.gfile.GFile(pb_model_path, 'wb') as f:
+        f.write(graph_def.SerializeToString())
+
+    # Convert to OpenVINO IR
+    subprocess.run(
+        [
+            sys.executable, mo_tf.__file__, '--input_model', pb_model_path,
+            '--input', ','.join(input_node_names), '--input_shape', "[1, 8912]",
+        ],
+        check=True)
+
+    roll_net = ie.read_network(model=model_xml, weights=model_bin)
+    roll_exec_net = ie.load_network(network=roll_net, device_name='CPU')
```

Comment:
Named arguments is a good practice but in this case variables names are meaningful enough. So let's avoid named arguments in these two lines.